common/src/main/java/tachyon/collections/IndexedSet.java

```diff
@@ -74,14 +74,14 @@
  * First, define the fields to be indexed:
  * <pre>
```

Comment:
`{@code ...}` is good for wrapping snippets of code which use generics, but doesn't work well with longer examples since any closing brace will end code-mode, and `<pre>` needs to be used anyway.

See the upvoted comments in http://stackoverflow.com/a/542142